index.bs

```diff
@@ -1278,6 +1278,8 @@ controls the default and maximum values that a source registration can specify
 for the epsilon parameter used by [=compute the channel capacity of a source=]
 and [=obtain a randomized source response=].
 
+<dfn>Max cardinality of possible trigger states</dfn> is a positive integer that controls the maximum size of [=obtain a set of possible trigger states|the set of possible trigger states=] for any one [=attribution source=].
```

Comment:
Why do we want to call max cardinatlity of this constant instead of just max possible trigger states?

Reply:
I think `maximum trigger states` is a good compromise.

Reply:
"maximum trigger states" doesn't seem accurate to me because I wouldn't think of "trigger states" as a number.

Reply:
`maximum number of trigger states`?

Reply:
My original thought was max number of report states. Is "trigger" preferred to "report." "Max" rather than maximum, seems consistent with the values around it.

Reply:
`max number of [whatever] states` is fine, where `[whatever]` is replaced with the terminology we use to describe the set.

Reply:
That terminology is exactly what I chose at the outset, from "obtain a set of possible trigger states." "Cardinality is a technical term for a set's size.

Reply:
OK. But the term `possible` doesn't add anything here. For brevity `max trigger-state cardinality` is fine.

Reply:
I'm fine with that.

Reply:
Done!

Reply:
I don't feel strongly, but we've been using "max ..." for a set's size throughout the spec, e.g. "max settable event-level report windows" and many others.